pkg/ccl/changefeedccl/event_processing.go

```diff
@@ -100,22 +121,45 @@ func newEventConsumer(
 			}
 		}
 
+		// Passing a nil Pacer is effectively a noop Pacer if
+		// CPU control is disabled.
+		var pacer *admission.Pacer = nil
+		if enablePacer {
+			tenantID, ok := roachpb.TenantFromContext(ctx)
+			if !ok {
+				tenantID = roachpb.SystemTenantID
+			}
+			pacer = cfg.PacerMaker.NewPacer(
+				pacerRequestUnit,
+				admission.WorkInfo{
```

Comment:
This workInfo struct should probably have some sort of a tag (an op-string or whatnot) identifying the queue. I know that at this point, pacer does not export per queue stats, but it probably should.
A todo would be fine (in the admission code)

Reply:
I'm worried about what would happen if you tag a `WorkInfo` with one queue's info, but send it to another. Also, maybe queues should collect stats instead of the `Pacer` type, because they are underlying mechanism of the `Pacer type. 

It may make more sense to tag `WorkInfo` with the type/origin of the work and have queues collect metrics about the work coming in. We can group queues by their [granter](https://github.com/cockroachdb/cockroach/blob/c8c85358743be9c1de1c8765352f7269f1d98ff1/pkg/util/admission/work_queue.go#L226-L227). For example, collect stats about cdc work being granted CPU time. @sumeerbhola, would you mind sharing your opinion about this?